src/main/java/org/everit/jira/timetracker/plugin/JiraTimetrackerChartWebAction.java

```diff
@@ -350,9 +351,7 @@ private boolean loadDataFromSession() {
     HttpSession session = getHttpSession();
     Object data = session.getAttribute(SESSION_KEY);
 
-    if (data == null) {
-      return false;
-    } else {
+    if ((data != null) && (data instanceof ReportSessionData)) {
```

Comment:
How about this:

if (data == null || !(data instanceof ReportSessionData)) {
  return false
}